src/Microsoft.AspNet.Diagnostics/ErrorPageOptions.cs

```diff
@@ -23,5 +25,11 @@ public ErrorPageOptions()
         /// source code referenced by the exception stack trace is present on the server.
         /// </summary>
         public int SourceCodeLineCount { get; set; }
+
+        /// <summary>
+        /// Provides files containing source code which is used to display the line(s) of exception.
```

Comment:
`Provides files containing source code used to display contextual information of an exception.`